engine/src/cubos/engine/plugins/renderer.cpp

```diff
@@ -0,0 +1,57 @@
+#include <cubos/engine/plugins/renderer.hpp>
+#include <cubos/engine/plugins/transform.hpp>
+#include <cubos/engine/plugins/window.hpp>
+
+#include <cubos/engine/ecs/draw_system.hpp>
```

Comment:
I think its best to remove `cubos/engine/ecs/draw_system.hpp` entirely and move the system definition here, like you did with the `draw` system (maybe rename `draw` to `render`). This way, the plugin implementation is hidden in its source file.

Reply:
Did that, but left the original `cubos/engine/ecs/draw_system.hpp` and `.cpp` files still in the repository, as there are other samples using them and I didn't want to break them.